### ISSUE TITLE
Not build with 11ty anymore?

### ISSUE THREAD

**Issue by reatlat** [11ty/11ty-community#765]
I recently checked out the leaderboard and noticed some projects built with Astro among the entries. While they are undoubtedly cool, they aren’t built with Eleventy.

- [ericcarlisle.com/](https://www.11ty.dev/speedlify/ericcarlisle-com/)
- [codesue.com/](https://www.11ty.dev/speedlify/codesue-com/)
- [carol.gg/](https://www.11ty.dev/speedlify/carol-gg/)
- [eventua11y.com/](https://www.11ty.dev/speedlify/eventua11y-com/)
- [laker.tech/](https://www.11ty.dev/speedlify/laker-tech/)
- [vegardit.com/en/](https://www.11ty.dev/speedlify/vegardit-com-en/)
- [bryanlrobinson.com/](https://www.11ty.dev/speedlify/bryanlrobinson-com/)
- [fabiomrbarbosa.com/](https://www.11ty.dev/speedlify/fabiomrbarbosa-com/)
- [francefinanceconsultants.fr/](https://www.11ty.dev/speedlify/francefinanceconsultants-fr/)
- [dino-fakten.de/](https://www.11ty.dev/speedlify/dino-fakten-de/)
- [ryzokuken.dev/](https://www.11ty.dev/speedlify/ryzokuken-dev/)

@zachleat are you agree?

**Comment by zachleat** [comment]
Yeah, for sure! Sites come and go—but they need to be removed from 11ty-community—not speedlify. They get pruned from speedlify upstream when we re-run the leaderboards.

**Comment by reatlat** [comment]
@zachleat , here is the list I found to remove from community repo

- [codesue.com/](https://www.11ty.dev/speedlify/codesue-com/)
- [eventua11y.com/](https://www.11ty.dev/speedlify/eventua11y-com/)
- [vegardit.com/en/](https://www.11ty.dev/speedlify/vegardit-com-en/)
- [bryanlrobinson.com/](https://www.11ty.dev/speedlify/bryanlrobinson-com/)
- [fabiomrbarbosa.com/](https://www.11ty.dev/speedlify/fabiomrbarbosa-com/)
- [francefinanceconsultants.fr/](https://www.11ty.dev/speedlify/francefinanceconsultants-fr/)


And from the next list is already pruned, but they are still presented on speedlify leaderboard... why?
- [ericcarlisle.com/](https://www.11ty.dev/speedlify/ericcarlisle-com/)
- [carol.gg/](https://www.11ty.dev/speedlify/carol-gg/)
- [laker.tech/](https://www.11ty.dev/speedlify/laker-tech/)
- [dino-fakten.de/](https://www.11ty.dev/speedlify/dino-fakten-de/)
- [ryzokuken.dev/](https://www.11ty.dev/speedlify/ryzokuken-dev/)


PR created #767
